commands/command.go

```diff
@@ -160,7 +171,7 @@ func listCommandCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	return nil
 }
 
-func deleteCommandCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+func archiveCommandCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	ok, response := c.DeleteCommand(args[0])
 	if response.Error != nil {
 		return errors.New("Unable to delete command '" + args[0] + "' error: " + response.Error.Error())
```

Comment:
Should this error message be changed to "Unable to archive command..." ?
@ethervoid   @mgdelacroix